hw6_epam_traning/src/main/java/com/mapLine/LineIsNull.java

```diff
@@ -0,0 +1,9 @@
+package com.mapLine;
+
+public class LineIsNull extends RuntimeException {
```

Comment:
Exception names should always have word "exception". For example: `LineIsNullException`
And I suggest to rename this exception like `EmptyLineException `or something like this